### PR TITLE
Format imports using goimports

### PR DESCRIPTION
**Pull request by wgliang**
**What this PR does / why we need it**:
/kind cleanup

Format imports using [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports)

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubelet/cm/cpumanager/cpu_manager.go [hunk 1]
```diff
@@ -27,13 +27,14 @@ import (
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 
+	"path"
```
Comment (CaoShuFeng) [inline pkg/kubelet/cm/cpumanager/cpu_manager.go]: This looks strange to me.
For me imports are splitted into three parts:
1. go standard lib, like "fmt", "sync".
2. third party lib, like "github.com/golang/glog"
3. k8s packages

Why "path" import is different from previous "fmt", "math", "sync" ?

Comment (wgliang) [reply pkg/kubelet/cm/cpumanager/cpu_manager.go]: Yeah,I think you found a bug of goimports, however, when I manually modified this, it didn't use any hints.

**Review by luxas — APPROVED** [review]
/approve

If we should do this though, please add it to our verification suite, so that code that isn't vetted by `goimports` can't be committed to the repo.
cc @kubernetes/sig-contributor-experience-pr-reviews FYI

**Comment by dixudx** [comment]
> If we should do this though, please add it to our verification suite, so that code that isn't vetted by goimports can't be committed to the repo.

@luxas Correct. I've filed #64078 to address this. PTAL

**Comment by cblecker** [comment]
/assign @deads2k @smarterclayton 
for approval

**Comment by wgliang** [comment]
/assign @deads2k @smarterclayton
for approval

**Comment by wgliang** [comment]
/cc @deads2k @smarterclayton @brendandburns @thockin @derekwaynecarr 
PTAL
It has been blocked for a long time.

**Comment by deads2k** [comment]
No strong feeling.

/approve 

if someone digs through for the details for an lgtm

File: pkg/cloudprovider/providers/vsphere/vsphere_util.go [hunk 2]
```diff
@@ -29,14 +29,15 @@ import (
 
 	"fmt"
 
-	"github.com/vmware/govmomi/vim25/mo"
 	"io/ioutil"
+	"path/filepath"
+
+	"github.com/vmware/govmomi/vim25/mo"
```
Comment (hzxuzhonghu) [inline pkg/cloudprovider/providers/vsphere/vsphere_util.go]: add a blank line

File: pkg/kubelet/cm/cpumanager/cpu_manager_test.go [hunk 3]
```diff
@@ -23,16 +23,17 @@ import (
 	"testing"
 	"time"
 
-	cadvisorapi "github.com/google/cadvisor/info/v1"
 	"io/ioutil"
+	"os"
+
+	cadvisorapi "github.com/google/cadvisor/info/v1"
```
Comment (hzxuzhonghu) [inline pkg/kubelet/cm/cpumanager/cpu_manager_test.go]: same here

File: pkg/volume/gce_pd/attacher_test.go [hunk 4]
```diff
@@ -27,10 +27,11 @@ import (
 	"k8s.io/kubernetes/pkg/volume"
 	volumetest "k8s.io/kubernetes/pkg/volume/testing"
 
+	"strings"
+
```
Comment (hzxuzhonghu) [inline pkg/volume/gce_pd/attacher_test.go]: This file is not well grouped

File: test/e2e/framework/timer/timer.go [hunk 5]
```diff
@@ -22,9 +22,10 @@ import (
 	"bytes"
 	"fmt"
 
+	"sync"
```
Comment (hzxuzhonghu) [inline test/e2e/framework/timer/timer.go]: remove blank line up

File: test/e2e/instrumentation/monitoring/stackdriver_metadata_agent.go [hunk 6]
```diff
@@ -25,13 +25,14 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	. "github.com/onsi/ginkgo"
 	"io/ioutil"
+	"reflect"
+
+	. "github.com/onsi/ginkgo"
```
Comment (hzxuzhonghu) [inline test/e2e/instrumentation/monitoring/stackdriver_metadata_agent.go]: here

File: test/integration/scheduler_perf/scheduler_test.go [hunk 7]
```diff
@@ -18,17 +18,18 @@ package benchmark
 
 import (
 	"fmt"
+	"math"
+	"strconv"
+	"testing"
+	"time"
+
 	"github.com/golang/glog"
```
Comment (hzxuzhonghu) [inline test/integration/scheduler_perf/scheduler_test.go]: add a blank line

Comment (wgliang) [reply test/integration/scheduler_perf/scheduler_test.go]: This is tool auto goimports formate.

**Comment by cblecker** [comment]
/lgtm cancel

@wgliang Can you please rebase this off of master, and re-run? I'm seeing that `pkg/cloudprovider/providers/vsphere/credentialmanager.go` also needs it's goimports fixed. Once this is done, I'm going to try to get this merged asap.

**Comment by wgliang** [comment]
@cblecker
Ok ,I'll rebase and re-run it for adding all new files.

**Comment by cblecker** [comment]
@wgliang I'm still seeing issues with:
```
+++ ./pkg/kubelet/types/pod_status_test.go      2018-05-31 10:10:32.000000000 -0700
+++ ./pkg/util/pod/pod_test.go  2018-05-31 10:10:37.000000000 -0700
+++ ./test/e2e/common/expansion.go      2018-05-31 10:10:54.000000000 -0700
```

**Comment by wgliang** [comment]
@cblecker 
Fix all, thanks!

**Comment by wgliang** [comment]
/cc @lavalamp @smarterclayton @thockin
PTAL,thanks.

**Comment by cblecker** [comment]
@wgliang Because of the rebases, this didn't end up making our 1.11 code freeze, and as such will have to wait until that's over. I don't want you to have to keep rebasing when there isn't a chance of it merging. Freeze will lift in 2 weeks. We should be able to re-prioritize this to get in at that time.

**Comment by wgliang** [comment]
@cblecker All right.

**Comment by wgliang** [comment]
/ping @smarterclayton @cblecker 
PTAL for lgtm and approve

File: pkg/volume/flexvolume/probe.go [hunk 8]
```diff
@@ -26,11 +26,12 @@ import (
 	"path/filepath"
 	"sync"
 
+	"strings"
```
Comment (dixudx) [inline pkg/volume/flexvolume/probe.go]: No need for a redundant line.

File: pkg/cloudprovider/providers/aws/instances.go [hunk 9]
```diff
@@ -19,15 +19,15 @@ package aws
...
 	"net/url"
+	"regexp"
 	"strings"
+	"sync"
+	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/golang/glog"
 	"k8s.io/api/core/v1"
```
Comment (dixudx) [inline pkg/cloudprovider/providers/aws/instances.go]: Normally we will group `k8s.io` packages together, keeping them separate from other non k8s.io packages. Right?

File: pkg/cloudprovider/providers/aws/regions.go [hunk 10]
```diff
@@ -17,10 +17,11 @@ limitations under the License.
 package aws
 
 import (
+	"sync"
+
 	"github.com/golang/glog"
```
Comment (dixudx) [inline pkg/cloudprovider/providers/aws/regions.go]: Ditto.

Add a newline here.

File: pkg/cloudprovider/providers/vsphere/credentialmanager.go [hunk 11]
```diff
@@ -19,13 +19,14 @@ package vsphere
 import (
 	"errors"
 	"fmt"
+	"net/http"
+	"strings"
+	"sync"
+
 	"github.com/golang/glog"
```
Comment (dixudx) [inline pkg/cloudprovider/providers/vsphere/credentialmanager.go]: Ditto. Add a newline here.

File: pkg/kubelet/cm/cpumanager/state/state_file.go [hunk 12]
```diff
@@ -19,11 +19,12 @@ package state
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/golang/glog"
 	"io/ioutil"
-	"k8s.io/kubernetes/pkg/kubelet/cm/cpuset"
 	"os"
 	"sync"
+
+	"github.com/golang/glog"
```
Comment (dixudx) [inline pkg/kubelet/cm/cpumanager/state/state_file.go]: Add a newline.

File: pkg/kubelet/cm/helpers_linux_test.go [hunk 13]
```diff
@@ -22,9 +22,10 @@ import (
 	"reflect"
 	"testing"
```
Comment (dixudx) [inline pkg/kubelet/cm/helpers_linux_test.go]: Remove newline.

File: pkg/kubelet/kuberuntime/security_context_test.go [hunk 14]
```diff
@@ -17,11 +17,11 @@ limitations under the License.
 package kuberuntime
 
 import (
-	"k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
 
 	"github.com/stretchr/testify/assert"
-	"testing"
+	"k8s.io/api/core/v1"
```
Comment (dixudx) [inline pkg/kubelet/kuberuntime/security_context_test.go]: Ditto. Add new line.

**Review by dixudx — CHANGES_REQUESTED** [review]
Normally we will group k8s.io packages together, keeping them separate from other non k8s.io packages.

**Comment by wgliang** [comment]
@dixudx DONE, thanks for your review.

File: pkg/volume/gce_pd/attacher_test.go [hunk 15]
```diff
@@ -19,18 +19,17 @@ package gce_pd
 import (
 	"errors"
 	"fmt"
+	"strings"
 	"testing"
 
+	"github.com/golang/glog"
```
Comment (dixudx) [inline pkg/volume/gce_pd/attacher_test.go]: Newline here.

File: test/e2e/instrumentation/monitoring/stackdriver_metadata_agent.go [hunk 16]
```diff
@@ -17,21 +17,20 @@ limitations under the License.
...
 	"context"
 	"encoding/json"
 	"fmt"
-	. "github.com/onsi/ginkgo"
 	"io/ioutil"
+	"reflect"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"golang.org/x/oauth2/google"
```
Comment (dixudx) [inline test/e2e/instrumentation/monitoring/stackdriver_metadata_agent.go]: Newline here.

File: test/e2e/storage/vsphere/config.go [hunk 17]
```diff
@@ -19,10 +19,11 @@ package vsphere
 import (
 	"errors"
 	"fmt"
-	"gopkg.in/gcfg.v1"
 	"io"
-	"k8s.io/kubernetes/test/e2e/framework"
 	"os"
+
+	"gopkg.in/gcfg.v1"
```
Comment (dixudx) [inline test/e2e/storage/vsphere/config.go]: Newline here.

File: test/e2e/storage/vsphere/vsphere_statefulsets.go [hunk 18]
```diff
@@ -18,6 +18,7 @@ package vsphere
 
 import (
 	"fmt"
+
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	apierrs "k8s.io/apimachinery/pkg/api/errors"
```
Comment (dixudx) [inline test/e2e/storage/vsphere/vsphere_statefulsets.go]: Newline here.

**Comment by dixudx** [comment]
@wgliang I find k8s.io packages are not well grouped in some other files. Like https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/kubelet/flags.go#L26.

So shall we group them as well.

**Comment by wgliang** [comment]
@dixudx Yes, this issue does exist, as you say. However, the goimports tool does not automatically recognize k8s.io and github.com and group them. So only manually group them or add features for goimports.

Anyway, I will try my best to group them manually, as you say. :-)

**Pull request closed without merging**